src/error_gui.cpp

```diff
@@ -75,11 +75,9 @@ static WindowDesc _errmsg_face_desc(
  * @param data The data to copy.
  */
 ErrorMessageData::ErrorMessageData(const ErrorMessageData &data) :
-	is_critical(data.is_critical), textref_stack_grffile(data.textref_stack_grffile), textref_stack_size(data.textref_stack_size),
-	summary_msg(data.summary_msg), detailed_msg(data.detailed_msg), extra_msg(data.extra_msg), position(data.position), company(data.company)
-{
-	memcpy(this->textref_stack, data.textref_stack, sizeof(this->textref_stack));
-}
+	is_critical(data.is_critical),
+	summary_msg(data.summary_msg), detailed_msg(data.detailed_msg), extra_msg(data.extra_msg),
+	position(data.position), company(data.company) {}
```

Comment:
Hasn't this now become a bog standard ` = default`-able copy-constructor? Or maybe even that can be left out now?

Reply:
Hmm, good point!